lib/reporters/spec.js

```diff
@@ -67,20 +67,34 @@ function Spec(runner, options) {
   });
 
   runner.on(EVENT_TEST_PASS, function (test) {
+    function logRetries() {
+      if (test._currentRetry > 0) {
+        /*
+          Included message within format so it can be passed into `Base.consoleLog` as one argument
+        */
+        var retryFmt = color(
+          'bright yellow',
```

Comment:
To really stick with the theme I think we should add a `'retries'` color here:

https://github.com/mochajs/mocha/blob/24560c1532b8e67d1254b489fcc6f84c4033c0e1/lib/reporters/base.js#L64-L90

And possibly stick even more with the style of eg. durations and resolve to the color in `base.js`:

https://github.com/mochajs/mocha/blob/24560c1532b8e67d1254b489fcc6f84c4033c0e1/lib/reporters/base.js#L369-L377

And maybe do a similar style and have two levels: `'some retries'` / `'many retries'` With only the latter resolving to bright yellow